src/hooks/UseRecentAds.ts

```diff
@@ -0,0 +1,17 @@
+import { Ads_URLS, axiosInstance } from "../services/urls";
+import useFetch from "./useFetch";
+
+export default function UseRecentAds() {
```

Comment:
It is more common for functions to start with small letters. Making it capitalized might make it confusing and other people using it might think that this is a component instead.